app/connector/debezium/src/main/java/io/syndesis/connector/debezium/DebeziumConsumerCustomizer.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.debezium;
+
+import java.io.IOException;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.syndesis.integration.component.proxy.ComponentProxyComponent;
+import io.syndesis.integration.component.proxy.ComponentProxyCustomizer;
+import org.apache.camel.Exchange;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DebeziumConsumerCustomizer implements ComponentProxyCustomizer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DebeziumConsumerCustomizer.class);
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    @Override
+    public void customize(ComponentProxyComponent component, Map<String, Object> options) {
+        component.setBeforeConsumer(DebeziumConsumerCustomizer::convertToDebeziumFormat);
+    }
+
+    private static void convertToDebeziumFormat(Exchange exchange) throws IOException {
+        String body = exchange.getMessage().getBody(String.class);
```

Comment:
How big are the message payloads, any chances they can be streamed instead of allocated on heap?